components/ResourceCard.tsx

```diff
@@ -27,20 +27,31 @@ export const ResourceCard = ({
       height="100%"
       direction="column"
       justify="space-between"
-    >
+      >
+      
       <Stack spacing="24px">
+        
         {resource.link ? (<Heading fontSize="2xl"><Link href={resource.link}>{resource.title}</Link></Heading>):(<Heading fontSize="2xl">{resource.title}</Heading>)}
       
         {resource.groupName && resource.groupName != NO_ASSOCIATED_GROUP && (
           <Text fontSize="xl" fontWeight="semibold" color="Gray.900">
             {resource.groupName}
           </Text>
         )}
+
+
         {resource.needs && (
           <Text fontSize="xl" fontWeight="semibold" color="Gray.900">
             {`Need: ${resource.needs[0]}`}
           </Text>
         )}
+
+        {resource.neighborhoodNames ? (<Text>{resource.neighborhoodNames}</Text>): null }
+
+        {/* {resource.streetAddress ? (<Link >{resource.streetAddress}</Link>): null } */}
+
+        {resource.streetAddress ? (<Link href={"https://www.google.com/search?q=" + resource.streetAddress} target="_blank" rel="noopener noreferrer" >{resource.streetAddress}</Link>): null }
```

Comment:
Would it make more sense to open the address up in Google Maps rather than just Google? Also not sure if there's an easy way to hook into the device's native maps app if it's a phone or tablet